src/core/ts/web-worker-scripts/translation-worker.js/getBergamotModelsForLanguagePair.spec.ts

```diff
@@ -107,6 +107,24 @@ describe("getBergamotModelsForLanguagePair", function() {
     assert.equal(downloadedModelFiles.length, 3);
   });
 
+  it("csen", async function() {
```

Comment:
can you refactor this, so that you don't need to copy paste code to add language pair? Imagine that you need to add 100 of them

Reply:
Let's refactor when we have more than a handful of languages, it's easy enough to do once we settle on a final testing setup. Keeping it repeated makes it easier to read.

Reply:
Filed: https://github.com/mozilla-extensions/bergamot-browser-extension/issues/220